p8e-api/src/main/kotlin/io/provenance/engine/service/ChaincodeInvokeService.kt

```diff
@@ -40,7 +43,7 @@ data class ContractRequestWrapper(
     val executionUuid: UUID,
     val request: MsgP8eMemorializeContractRequest,
     val future: CompletableFuture<ContractTxResult>,
-)
+    )
```

Comment:
Is your IDE automatically making these format changes?

Reply:
Uhhhhh yeah. Totally. For sure. 😅
I assume I did some odd stuff during merge of os-locator or when I was removing some of the code I had in there for initial testing. Will make sure it doesn't happen again.